.github/actions/pnpm/install-binding-dependencies/action.yml

```diff
@@ -31,7 +31,7 @@ runs:
       shell: bash
       run: |
         cd ./crates/node_binding
-        pnpm install --ignore-workspace --no-lockfile
```

Comment:
The bad part is we linked `binding-builder-cli` to `crates/node_binding`. This needs us to fallback to workspace install instead of plain package install. cc @stormslowly